doc/internal/README-multiprotocol.md

```diff
@@ -15,7 +15,7 @@ potentially confronted with packets encoded in different schemas like
 ccnb, CCNx2014 or NDN2013. In the past it was possible to discriminate
 among these packets because luckily enough the "type values" were
 sparsely populated and the programmers had chosen code points that do
-not overlap. But with the compact IOT2014 format, there is no sparsity
+not overlap. But with no packet formats, there might be no sparsity
```

Comment:
>But with no packet formats, there might be no sparsity

Hmm, this sounds weird now ..

Maybe just remove this block:
>In the past it was possible to discriminate among these packets because luckily enough the "type values" were sparsely populated and the programmers had chosen code points that do not overlap. But with no packet formats, there might be no sparsity anymore. 

?

Reply:
That's right. It was supposed to be "new" and not "no".